.gitignore

```
@@ -1,7 +1,6 @@
 .idea/
 
 /.yarn/*
```

Let's revert this since it doesn't have consensus
```suggestion
/.yarn/*
!/.yarn/cache
```